CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
   the `master` `Device` occupies a `vc_position` other than `1`.
 - #43 - Development and CI testing now defaults to Nautobot 1.0.0 instead of 1.0.0b3
 - #43 - Fix test approach to ensure that tests execute against the test database rather than the development database.
+- #47 - ChangeLogged objects honour `created` date and also ChangeObject to represent the migration is created
```

Comment:
Please add a new release entry (I'm fine with calling this `## 1.4.0 (2021-05-xx)` for now) rather than adding this under the existing 1.3.0 release.

Reply:
ok, I thought this was the latest 🤦